mycore.org/content/de/documentation/migrate/migrate_mcr2025_12.html

```diff
@@ -58,6 +58,55 @@ <h3>... die eine Migration erforderlich machen</h3>
 
       <h2>Migrationsschritte</h2>
       <ul>
+        <li>
+          <p>
+            Bei allen MyCoRe-Modulen wurden die Tests von JUnit 4 auf JUnit 5 umgestellt.
+            Die und Unterstützung für JUnit 4 wurde entfernt. Dementsprechend sind auch die Test-Basisklassen
+            <code>MCRTestCase</code>, <code>MCRJPATestCase</code> und <code>MCRStoreTestCase</code> entfernt worden.
+          </p>
+          <ul>
+            <li>
+              Statt <code>MCRTestCase</code> zu erweitern, kann die Annotation <code>@MyCoReTest</code> verwendet werden.
+            </li>
+            <li>
+              Statt <code>MCRJPATestCase</code> zu erweitern, kann zusätzlich zu <code>@MyCoReTest</code> die Annotation<br/>
+              <code>@ExtendWith(MCRJPAExtension.class)</code> verwendet werden.
+            </li>
+            <li>
+              Statt <code>MCRStoreTestCase</code> zu erweitern, kann je nach Funktionsbedarf zusätzlich zu <code>@MyCoReTest</code> die Annotation<br/>
+              <code>@ExtendWith(MCRStoreExtension.class)</code> oder <code>@ExtendWith({MCRJPATestExtension, MCRStoreExtension.class})</code><br/>
+              verwendet werden.
+            </li>
+          </ul>
+          <br/>
+          <p>
+            Statt <code>getTestProperties()</code> zu überschreiben, kann mit <code>@MCRTestConfiguration</code> 
+            oder, wenn die Werte dynamisch sind, direkt mit <code>MCRConfiguration2#set</code> gearbeitet werden.
+          </p>
+          <p>Bestehende <code>setUp</code>- und <code>tearDown</code>-Methoden können erhalten bleiben, müssen aber als 
+            <code>static</code> markiert werden und mit <code>@BeforeAll</code> bzw. <code>@AfterAll</code> annotiert werden.
+            Der Aufruf der jeweiligen Super-Methode sowie eine ggf. vorhandene <code>@Override</code>-Annotation kann
+            ersatzlos entfernt werden.
+          </p>
+          <p>
+            Wer seine Tests nicht direkt während der Migration umstellen möchte, kann das neue Modul
+            <code>mycore-junit4</code>, in dem die Unterstützung für JUnit 4 vorerst erhalten bleibt,
+            zusammen mit JUnit 4 einbinden:
+            {{< highlight xml "linenos=table">}}
+            <dependency>
+              <groupId>junit</groupId>
+              <artifactId>junit</artifactId>
+              <version>4.13.2</version>
+              <scope>test</scope>
+            </dependency>
+            <dependency>
```

Comment:
Macht es ggf. Sinn, die Junit-Abhängigkeit in der pom von `mycore-junit4` zu definieren?